modules/_labs/combobox/react/lib/Status.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {useState, useEffect} from 'react';
+import styled from '@emotion/styled';
+import {accessibleHide} from '@workday/canvas-kit-react-common';
+
+export interface StatusProps {
+  ariaLive?: 'polite' | 'assertive' | 'off';
+  role?: 'log' | 'status' | 'alert' | 'progressbar' | 'marquee' | 'timer';
+  ariaRelevant?: 'additions' | 'additions text' | 'all' | 'removals' | 'text';
+  ariaAtomic?: boolean;
+  expireMilliseconds?: number;
+  announcementText?: string;
+}
+
+const Container = styled('div')({
+  ...accessibleHide,
+});
+
+const Status = ({
+  ariaLive = 'polite',
+  role = 'log',
+  ariaAtomic = false,
+  ariaRelevant = 'additions',
+  expireMilliseconds = 500,
+  announcementText = '',
+}: StatusProps) => {
+  const [displayText, setDisplayText] = useState(announcementText);
+  useEffect(() => {
+    if (!announcementText) {
+      return;
+    }
+    setDisplayText(announcementText);
+    const clearText = setTimeout(() => setDisplayText(''), expireMilliseconds);
+
+    return () => {
+      clearTimeout(clearText);
+    };
```

Comment:
could be
`return () => clearTimeout(clearText)`
clearTimeout returns void

Reply:
Same as above, I kinda like making it easy to add things to the return in the future. Happy to change if folks feel strongly.

Reply:
its fine! :) I am just afraid of the word "future" :D No one can possible predict it